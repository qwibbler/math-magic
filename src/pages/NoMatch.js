import React from 'react';
import Header from '../components/Header';

const NoMatch = () => (
  <section className="home">
    <Header />
    <main>
      <h2>
        Page Not Found.
      </h2>
    </main>
  </section>
);
export default NoMatch;
