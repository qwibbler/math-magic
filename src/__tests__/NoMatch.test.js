import React from 'react';
import NoMatch from '../pages/NoMatch';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';

test('NoMatch', () => {
  const tree = renderer
    .create(
      <Router>
        <NoMatch />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
