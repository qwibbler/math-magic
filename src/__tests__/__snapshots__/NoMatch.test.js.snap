// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch 1`] = `
<section
  className="noMatch"
>
  <header>
    <h1>
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M313.203 25.94l-.133 24.67-18.12 16.743 24.67.132 16.743 18.12.133-24.67 18.12-16.743-24.67-.132-16.743-18.12zm91.58 50.12L385.47 91.41l-24.4-3.638 15.35 19.313-3.637 24.4 19.313-15.35 24.4 3.637-15.35-19.312 3.637-24.4zM39.87 80.206l-7.116 16.268 7.637 3.34 256 112 8.247 3.607-.1.227.61.266 7.214-16.492-8.243-3.607-44.113-19.298a102.607 43.83 0 0 1 52.088-6.13A102.607 43.83 0 0 1 414.7 214.22a102.607 43.83 0 0 1-102.608 43.83 102.607 43.83 0 0 1-102.608-43.83 102.607 43.83 0 0 1 8.262-17.162l-32.814-14.358c-12.552 5.49-22.514 11.71-29.192 17.96-7.646 7.16-10.802 13.782-10.802 20.134s3.156 12.975 10.802 20.133c7.646 7.158 19.596 14.274 34.81 20.3 30.43 12.047 73.71 19.835 121.542 19.835 47.832 0 91.11-7.788 121.54-19.836 15.215-6.025 27.165-13.14 34.81-20.3 7.647-7.157 10.804-13.78 10.804-20.132 0-6.352-3.157-12.975-10.803-20.133-7.646-7.16-19.596-14.275-34.81-20.3-30.43-12.048-73.71-19.836-121.54-19.836-26.422 0-51.44 2.386-73.683 6.543L48.117 83.813l-8.246-3.607zm204.903 4.31l.077 16.708-12.16 11.46 16.708-.076 11.46 12.16-.075-16.708L272.94 96.6l-16.706.075-11.46-12.16zM215.94 286.592c-.147 15.018-.44 29.74-1.038 44.308 67.544 29.705 126.598 29.746 194.383.113-.6-14.603-.894-29.364-1.043-44.42-27.33 7.925-60.396 12.467-96.15 12.467-35.756 0-68.82-4.54-96.15-12.467zm-2.004 63.467c-.555 8.977-1.254 17.925-2.13 26.877l.555.25c69.454 31.175 129.756 31.177 199.486-.006l.535-.238c-.874-8.92-1.572-17.838-2.126-26.785-66.975 27.646-129.55 27.616-196.318-.097zm200.552 45.595c-70.003 30.062-135.025 30.067-204.793.002-2.287 17.913-5.385 35.95-9.603 54.403 0 48 224 48 224 0-4.218-18.455-7.316-36.49-9.604-54.405z"
        />
      </svg>
      Math Magicians
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M237.4 20.73c-6.1 42.1-26.8 64.2-63.9 64 31.6 4.5 63.8 8 63.9 64.07-.6-46.1 24.5-63.07 64.1-64.07-38-1.5-64.9-16.3-64.1-64zm127.8 11.58c-9.1 14.25-20.8 21.29-38.9 10.28 14.9 11.79 18.6 24.76 10.2 38.97 8.9-11.18 17.5-22.73 39-10.27-17.8-10.06-18.8-23.57-10.3-38.98zM59.68 41.69c-2.7 18.8-12 28.6-28.5 28.5 14.1 2 28.4 3.6 28.5 28.52-.3-20.5 10.9-28.12 28.5-28.52-16.9-.7-28.9-7.3-28.5-28.5zM431 66.28c-2.7 18.8-12 28.6-28.5 28.5 14.1 2 28.4 3.6 28.5 28.52-.3-20.5 10.9-28.12 28.5-28.52-16.9-.7-28.9-7.3-28.5-28.5zM120.3 116.4c-15.8 53.7-47.76 48-79.35 43.4C76.6 170 90.3 197.1 84.28 239.2c12.66-46 42.62-52.6 79.42-43.4-37.6-12.1-56.9-35.4-43.4-79.4zm187 5c-8.8 61.6-39.3 94-93.6 93.7 46.2 6.5 93.6 11.7 93.6 93.7-.8-67.3 35.9-92.2 93.8-93.7-55.5-2.2-94.9-23.9-93.8-93.7zm136.8 38.3c-13.1 21.6-29.5 28.8-49.7 20.1 16.3 9.7 33 19.1 20.1 49.6 10.3-25.2 27.9-28.7 49.7-20-20.3-9.7-31.6-23.9-20.1-49.7zM50.7 243.2c9.16 16.7 7.63 30.1-5.61 40 12.46-6.9 24.85-14.3 39.91 5.6-12.57-16.2-8.2-29 5.61-40-13.92 9.7-27.47 11.6-39.91-5.6zm137.2.3c11.4 26.8-.5 41.3-21.7 50.9 22.7-8.5 40.8-4.5 50.9 21.7-12.7-31.8 4.8-41.2 21.7-50.9-21 8.5-37.8.9-50.9-21.7zm228 12.6c-26.6 64.7-68.7 91.7-127.8 76.4 48.6 19.8 98.8 38.5 76.4 127.9 17.5-73.7 64.4-90.7 127.9-76.5-59.9-17.5-96.9-52-76.5-127.8zM99.94 295.5c15.66 57.8.86 98.1-47.32 118.5 43.46-11.8 87.38-25.2 118.68 47.4-26.4-59.3-3.4-95.4 47.3-118.8-50 19.2-93.1 15-118.66-47.1zm169.36 61c-21.8 20.6-43 23.6-63.2 7.3 15.5 16.3 31.6 32.4 7.2 63.3 19.8-25.6 41.2-24.1 63.3-7.3-20.2-17.4-28.6-37.5-7.3-63.3zM443.2 404c-2.7 18.8-12 28.6-28.5 28.5 14.1 2 28.4 3.6 28.5 28.5-.3-20.5 10.9-28.1 28.5-28.5-16.9-.7-28.9-7.3-28.5-28.5zm-169.7 36c-2.7 18.8-12 28.6-28.5 28.5 14.1 2 28.4 3.6 28.5 28.5-.3-20.5 10.9-28.1 28.5-28.5-16.9-.7-28.9-7.3-28.5-28.5z"
        />
      </svg>
    </h1>
    <nav>
      <a
        href="#/"
        onClick={[Function]}
      >
         Home 
      </a>
      |
      <a
        href="#/calculator"
        onClick={[Function]}
      >
         Calculator 
      </a>
      |
      <a
        href="#/quote"
        onClick={[Function]}
      >
         Quote 
      </a>
    </nav>
  </header>
  <main>
    <h2>
      Page Not Found.
    </h2>
  </main>
</section>
`;
